coordinator/clientapi/clientapi_test.go

```diff
@@ -164,51 +130,59 @@ func TestGetCertQuote(t *testing.T) {
 			defer log.Sync()
 
 			api := &ClientAPI{
-				core: tc.core,
-				data: tc.storeWrapper,
-				log:  log,
+				core:     tc.core,
+				txHandle: tc.store,
+				log:      log,
+			}
+
+			var intermediateCert, rootCert *x509.Certificate
+			if !tc.wantErr {
+				intermediateCert = testutil.GetCertificate(t, tc.store, constants.SKCoordinatorIntermediateCert)
+				require.NoError(err)
+				rootCert = testutil.GetCertificate(t, tc.store, constants.SKCoordinatorRootCert)
+				require.NoError(err)
```

Comment:
```suggestion
				intermediateCert = testutil.GetCertificate(t, tc.store, constants.SKCoordinatorIntermediateCert)
				rootCert = testutil.GetCertificate(t, tc.store, constants.SKCoordinatorRootCert)
```
and some other places where you switched to using testutil